horace_core/sqw/@sqw/resolution_plot.m

```diff
@@ -0,0 +1,231 @@
+function varargout = resolution_plot (w, varargin)
```

Comment:
should this be in the plot folder, or is this transitory? It has detailed knowledge of specific instruments that don't belong in the core SQW class

Reply:
At the moment the detailed instrument stuff still exists in @sqw i.e. methods tobyfit_DG${inst}_*. It would be better to move all that out systematically when the conversion of sqw_old to sqw is complete.